presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeGeneralQueries.java

```diff
@@ -42,6 +42,10 @@ protected void assertQuery(String sql)
     }
 
     // TODO: Enable following Ignored tests after fixing (Tests can be enabled by removing the method)
+    @Override
+    @Ignore
+    public void testUnionAll() {}
```

Comment:
The failure in the GitHub issue is generic and I'd expect it to happen in both Prestissimo and Presto-on-Spark tests. Let's wait for CI to get fixed before disabling tests.